### PR TITLE
Avoid using translations

### PR DESCRIPTION
**Pull request by adeas31**
Workaround for #130

### CODE REVIEW

File: OMPython/__init__.py [hunk 1]
```diff
@@ -698,6 +698,7 @@ def __init__(self, readonly=False, timeout = 10.00, docker = None, dockerContain
         # set omc executable path and args
         self._set_omc_command([
                                "--interactive=zmq",
+                               "--locale=C",
```
Comment (sjoelund) [inline OMPython/__init__.py:701]: This won't work with some older OMC versions

Comment (adeas31) [reply OMPython/__init__.py:701]: Are you talking about like 1.12 or later. Anyway people using such old OMC can use older pip install version of OMPython.

Comment (bilderbuchi) [reply OMPython/__init__.py:701]: then that information concerning the minimum OMC version should probably be put somewhere (Readme, setup.py, warning at import time?)

Comment (sjoelund) [reply OMPython/__init__.py:701]: I load 1.9 versions of OMC that are older than this, and they work just fine... For zmq you could put it, but I guess not CORBA

**Comment by adeas31** [comment]
Since the translations are removed from OMC so this issue will not be seen in the future versions anyway. One idea is to close this PR. What do you think @sjoelund? Shall I close or merge this?

**Comment by casella** [comment]
RIP multilingual support. I always switched to English anyway...

**Pull request merged**